protocol/x/clob/keeper/grpc_stream_orderbook.go

```diff
@@ -18,3 +20,54 @@ func (k Keeper) StreamOrderbookUpdates(
 	<-ctx.Done()
 	return nil
 }
+
+// revertOptimisticFills sends OrderUpdates that effectively reverts the optimistic fills.
+func (k Keeper) revertOptimisticFills(ctx sdk.Context) {
+	// Send absolute fill amounts from local to the grpc stream.
+	// This must be sent out to account for checkState being discarded and deliverState being used.
+	localValidatorOperationsQueue, _ := k.MemClob.GetOperationsToReplay(ctx)
+	orderIdsFromLocalOpsQueue := fetchOrdersInvolvedInOpQueue(
```

Comment:
No longer fetching orders from proposer ops queue, which from offline discussion we agree isn't really needed